documentation/commands/ship.md

```diff
@@ -31,6 +31,12 @@ GitHub API. Setup:
    squash merge via the GitHub API. It will also update the base branch for any
    pull requests against that branch.
 
+If you use
+[GitHub's feature to automatically delete head branches](https://help.github.com/en/github/administering-a-repository/managing-the-automatic-deletion-of-branches),
+set `git-town.delete-remote-branches` to `false` in your local or global Git
```

Comment:
Thoughts on naming this `git-town.ship-delete-remote-branch`. Like the ship prefix as it only effects this command and don't think this needs to be plural as a single invocation will only effect one branch

Reply:
I like it! Renamed.